modules/lx-instance/watchmaker-lx-instance.cfn.json

```diff
@@ -323,6 +336,13 @@
             "Value": {
                 "Ref": "WatchmakerInstance"
             }
+        },
+        "WatchmakerInstanceLogGroupName": {
+            "Condition": "InstallCloudWatchAgent",
+            "Description": "Log Group Name",
+            "Value": {
+                "Ref": "WatchmakerInstanceLogGroup"
+            }
```

Comment:
I have the log group name in the `Outputs` section.

Reply:
:shameface: